6. Developer Guides/REST API/Im/messages.others.md

```diff
@@ -0,0 +1,88 @@
+---
+order: 25
+---
+
+# Im Messages.others
+Retrieves the messages from any direct message in the server, this method supports the [Offset and Count Query Parameters](../Offset%20and%20Count%20Info.md).
+
+For this method work the `Enable Direct Message History Endpoint` setting must be true, and the user calling this method must have the `view-room-administration` permission.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/im.messages.others` | `yes` | `GET` |
+
+## Payload
```

Comment:
@graywolf336  Fixed! Also fixed the other errors like this one in some other files